config/_default/config.yml

```diff
@@ -11,6 +11,13 @@ module:
           target: content
         - source: static
           target: static
+          excludeFiles:
```

Comment:
This is an optimization to make `hugo server` not watch for changes to dependencies of example programs.